packages/cli/src/util/telemetry/commands/integration/list.ts

```diff
@@ -10,7 +15,7 @@ export class IntegrationListTelemetryClient extends TelemetryClient {
     }
   }
 
-  trackCliOptionIntegration(v: string | undefined, known: boolean | undefined) {
+  trackCliOptionIntegration(v: string | undefined, known?: boolean) {
```

Comment:
Why does this type change?

Reply:
The type from the generic interface is `trackCliOptionIntegration(v: string | undefined)` (single parameter), so in TypeScript any additional params must be optional so that the original interface remains conformant.

Reply:
Ok, that `...rest` trick _does_ work, and allows for additional, required, parameters to be defined. The only downside is that when auto implementing the interface, the editor includes the `...rest` parameter in the function signature (which will be deleted most of the time):

```
  trackCliArgumentName: (value: string | undefined, ...rest: any[]) => void;
```

So while that does work, I'm kind of ±0 on the change - not sure if the DX of "fixing up" that `...rest` parameter is a worthy tradeoff. But I could easily have my mind changed about that if we can come up with an example of when a required 2nd parameter would make sense.